src/pipeline.ts

```diff
@@ -270,17 +282,20 @@ export async function searchForLocalTorrentByCriteria(
 
 export async function checkNewCandidateMatch(
 	candidate: Candidate,
+	candidateLog: string,
```

Comment:
it feels weird to pass around a log string rather than richer, structured, information. Is this because `candidate.tracker` doesn't exist?